src/Pages/Overview/OverviewProfile.js

```diff
@@ -69,35 +70,19 @@ export default function OverviewProfile (props) {
         <InfoCard user={props.user} token={props.token} />
       </Modal>
 
-      <Modal
-        isOpen={toggleDelete}
+      <ConfirmationModal
+        headerText={`Delete ${props.user.name} ?`}
```

Comment:
the modal looks like this when we run it
![image](https://user-images.githubusercontent.com/36345325/75586472-58e84f00-5a29-11ea-9743-ee9b6113b153.png)